Document/0x04f-Testing-Network-Communication.md

```diff
@@ -431,77 +485,73 @@ $ iptables -t nat -A POSTROUTING -j MASQUERADE
+### Setting Up Traffic Redirection and the Interception Proxy
 
-1. Go to **Proxy** tab and click on **Options**
-2. Select and edit your listener from the list of proxy listeners.
-3. Go to **Request handling** tab and set:
+If you haven't already, [set up your interception proxy](#mastg-tech-intercepting-http-traffic-using-an-interception-proxy).
 
-    - Redirect to host: provide original traffic location.
-    - Redirect to port: provide original port location.
-    - Set 'Force use of SSL' (when HTTPS is used) and set 'Support invisible proxy'.
+**Tip:** When redirecting traffic, create specific rules for only the domains and IPs in scope to reduce noise from out-of-scope traffic.
 
-<img src="Images/Chapters/0x04f/burp_xamarin.png" width="100%" />
+Ensure that your interception proxy listens on the port defined in your redirection rule (`8080` in this case).
 
-#### CA Certificates
+After redirecting traffic to your interception proxy, you need to forward it back to its original destination. The following steps set up redirection in @MASTG-TOOL-0077:
 
-If not already done, install the CA certificates in your mobile device which will allow us to intercept HTTPS requests:
+1. Open the **Proxy** tab and click on **Options**.
+2. Select and edit your listener from the list of proxy listeners.
+3. Navigate to the **Request Handling** tab and configure:
+    - **Redirect to host**: Set this to the original server destination.
+    - **Redirect to port**: Specify the original port.
+    - Enable **"Force use of SSL"** (if HTTPS is used) and **"Support invisible proxy"**.
 
-- [Install the CA certificate of your interception proxy into your Android phone](https://support.portswigger.net/customer/portal/articles/1841102-installing-burp-s-ca-certificate-in-an-android-device "Installing Burp\'s CA Certificate in an Android Device")
-    > Note that starting with Android 7.0 (API level 24) the OS no longer trusts a user supplied CA certificate unless specified in the app. Bypassing this security measure will be addressed in the "Basic Security Testing" chapters.
-- [Install the CA certificate of your interception proxy into your iOS phone](https://support.portswigger.net/customer/portal/articles/1841108-configuring-an-ios-device-to-work-with-burp "Configuring an iOS Device to Work With Burp")
+<img src="Images/Chapters/0x04f/burp_xamarin.png" width="100%" />
 
-#### Intercepting Traffic
+### Start Intercepting Traffic
 
-Start using the app and trigger its functions. You should see HTTP messages showing up in your interception proxy.
+Now, start using the app and trigger its functions. If configured correctly, HTTP messages should appear in your interception proxy.
 
-> When using bettercap you need to activate "Support invisible proxying" in Proxy Tab / Options / Edit Interface
+> **Note:**  
+> When using Bettercap, enable **"Support invisible proxying"** under **Proxy Tab → Options → Edit Interface**.  
```

Comment:
Also necessary with DNS spoofing, because the client doesn't know that it's being intercepted.